mne/epochs.py

```diff
@@ -1556,13 +1556,10 @@ def equalize_event_counts(self, event_ids, method='mintime', copy=True):
             If 'truncate', events will be truncated from the end of each event
             list. If 'mintime', timing differences between each event list
             will be minimized.
-        copy : bool
-            If True, a copy of epochs will be returned. Otherwise, the
-            function will operate in-place.
```

Comment:
Also Teon asked for a fix, I may be misunderstanding what the actual problem was.